HW3/Training-HW3-flex/index.html

```diff
@@ -5,51 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Training HW3</title>
-    <link rel="stylesheet" href="css/styles.css" />
     <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
     <div class="container">
-      <div class="bubble">
-        <span style="--i: 11"></span>
-        <span style="--i: 12"></span>
-        <span style="--i: 24"></span>
-        <span style="--i: 10"></span>
-        <span style="--i: 14"></span>
-        <span style="--i: 23"></span>
-        <span style="--i: 18"></span>
-        <span style="--i: 16"></span>
-        <span style="--i: 19"></span>
-        <span style="--i: 20"></span>
-        <span style="--i: 22"></span>
-        <span style="--i: 14"></span>
-        <span style="--i: 18"></span>
-        <span style="--i: 25"></span>
-        <span style="--i: 29"></span>
-        <span style="--i: 17"></span>
-        <span style="--i: 13"></span>
-        <span style="--i: 16"></span>
-        <span style="--i: 10"></span>
-        <span style="--i: 22"></span>
-        <span style="--i: 25"></span>
-        <span style="--i: 13"></span>
-        <span style="--i: 16"></span>
-        <span style="--i: 10"></span>
-        <span style="--i: 22"></span>
-        <span style="--i: 21"></span>
-        <span style="--i: 14"></span>
-        <span style="--i: 18"></span>
-    </div>
-      <header class="heading">
-        <div class="heading__name">
-          <a href="#"
-            ><img src="img/icons8-milan-46.png" alt="Milan Holidays"
-          /></a>
-          <div class="heading__title">
-            <a href="#"><h2>MILAN</h2></a>
-            <a href="#"><h3>HOLIDAYS</h3></a>
+      <header class="heading-container">
+        <a href="#" class="logo">
+          <div class="milan-holidays"></div>
+          <div class="logo__header">
```

Comment:
please take a look one more time on the logo animation
![anim](https://user-images.githubusercontent.com/44003977/218421278-03c5465e-c0e7-4322-99c7-123325a20db8.gif)